modules/seqexec/model/src/test/scala/seqexec/model/arb/ArbCommaOption.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.model.arb
+
+import org.scalacheck.Arbitrary
+import org.scalacheck.Cogen
+import org.scalacheck.Gen
+import seqexec.model.enum.ComaOption
+
+trait ArbComaOption {
```

Comment:
It seems like all of these enums should provide `gem.util.Enumerated` instances.  If they did, you'd get the arbitraries and cogens for free from `gem.arb.ArbEnumerated`.

Reply:
Hmm. That sounds good, I guess we can use that even for non generated enums?